.chloggen/splunkent-client-refactor.yaml

```diff
@@ -1,20 +1,27 @@
 # Use this changelog template to create an entry for release notes.
-# If your change doesn't affect end users, such as a test fix or a tooling change,
-# you should instead start your pull request title with [chore] or use the "Skip Changelog" label.
 
 # One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
 change_type: 'enhancement'
 
 # The name of the component, or a single word describing the area of concern, (e.g. filelogreceiver)
-component: splunkentreceiver
+component: splunkentreceiver 
 
 # A brief description of the change.  Surround your text with quotes ("") if it needs to start with a backtick (`).
-note: "adding component logic to splunkenterprise receiver"
+note: Refactor Splunkenterprise Receiver component to better leverage existing otel SDK.
```

Comment:
Can you create a separate changelog for this? You sure are going to run into conflicts when we release otherwise.

Reply:
yea i have no idea why this is being tracked like this. I guess 'cause I forked a previously unmerged branch?

Reply:
no, this is from your last PR. The .chloggen tool will take all the .yaml files and merge them into CHANGELOG.md at release - right about now. Then it deletes all .yaml files and we continue. So if you make 2 changes in one release, best to have 2 changelog entries.